datus-clickzetta/scripts/init_tpch_data.py

```
@@ -0,0 +1,263 @@
+            insert_result = conn.execute_insert(data)
+            if not insert_result.success:
+                print(f"  Failed inserting into {TPCH_TABLES[i]}: {insert_result.error}")
+                sys.exit(2)
+            print(f"  Inserted {ROW_COUNTS[i]} rows into {TPCH_TABLES[i]}")
+
+        # Verify
+        print("\nVerifying data...")
+        has_mismatch = False
+        for i, table in enumerate(TPCH_TABLES):
+            result = conn.execute_query(
+                f"SELECT COUNT(*) AS cnt FROM `{table}`",
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)